### PR TITLE
fix: fail install on missing csr-signer secret

### PR DESCRIPTION
**Pull request by Roming22**

### CODE REVIEW

**Review by otaviof — APPROVED** [review]
Just a `nit` comment, but looks good. Thanks!

File: hack/fix-csr-signer.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
```
Comment (otaviof) [inline hack/fix-csr-signer.sh]: Would be good to have a comment header explaining why this scripts exists.

Comment (Roming22) [reply hack/fix-csr-signer.sh]: I'm having issues with the workaround. Since it's not critical for the release, I removed it from this PR.

**Comment by Roming22** [comment]
#199 is the right fix

**Pull request closed without merging**